ide/html.editor/src/org/netbeans/modules/html/editor/hints/HtmlHintsProvider.java

```diff
@@ -166,6 +166,7 @@ public void computeSuggestions(HintsManager manager, RuleContext context, List<H
         if(ExtractInlinedStyleRule.SINGLETON.appliesTo(context)) {
             suggestions.add(new ExtractInlinedStyleHint(context, new OffsetRange(context.caretOffset, context.caretOffset)));
         }
+
```

Comment:
This white-space change should not be here.

Reply:
Just a leftover after I removed my check. Will remove it.